<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EN61850monbase</class>
 <widget class="QDialog" name="EN61850monbase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>375</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>EN61850-9-2 Monitor</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="Q3ButtonGroup" name="MonitorCountbuttonGroup">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Counters</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="ByteCountLabel">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Bytes received:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="ByteCountValLabel">
          <property name="text">
           <string>0</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="spacer36">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>210</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="LostSyncCountLabel">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Syncronization lost: </string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="LostSyncValLabel">
          <property name="text">
           <string>0</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="spacer37">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>130</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="Q3ButtonGroup" name="MACErrorsbuttonGroup">
       <property name="title">
        <string>MAC Errors</string>
       </property>
       <layout class="QVBoxLayout">
        <item>
         <layout class="QHBoxLayout">
          <item>
           <layout class="QVBoxLayout">
            <item>
             <widget class="QCheckBox" name="RUNTFramecheckBox">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="toolTip">
               <string/>
              </property>
              <property name="text">
               <string>Runt Frame</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="LateCollisioncheckBox">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Late Collision</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="RWTOcheckBox">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Receive Timeout</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="PHysLayercheckBox">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Physical Layer</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout">
            <item>
             <widget class="QCheckBox" name="AlignErrorcheckBox">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Alignment</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="CRCErrorcheckBox">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>CRC</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="FIFOOvfcheckBox">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>FIFO Overflow</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="MACSyncLostcheckBox">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Sync lost</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="spacer5">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>16</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="Q3ButtonGroup" name="DecoderErrorbuttonGroup">
       <property name="title">
        <string>Decoder Errors</string>
       </property>
       <layout class="QHBoxLayout">
        <item>
         <layout class="QVBoxLayout">
          <item>
           <widget class="QCheckBox" name="savPducheckBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>savPdu</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="ASDUcheckBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>num. of ASDU</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="seqASDUcheckBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>seq. of ASDU's</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="seqASDUncheckBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>seq. of ASDU n</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="serviceIdentcheckBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Service Ident.</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="smpCountcheckBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>smpCnt</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="confRevcheckBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>confRev</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout">
          <item>
           <widget class="QCheckBox" name="smpSynchcheckBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>smpSynch</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="seqDatacheckBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>seq. of Data</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="ETHMacAdrcheckBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>MAC Adress mismatch</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="ETHHeadercheckBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Ethertype, APPID</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="PriorityTaggedcheckBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Priority tagged</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="nASDUcheckBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>ASDU not avail</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="DataSyncLostcheckBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Data Sync lost</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <spacer name="Horizontal Spacing2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>300</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonOk">
       <property name="text">
        <string>&amp;OK</string>
       </property>
       <property name="shortcut">
        <string/>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel">
       <property name="text">
        <string>&amp;Reset</string>
       </property>
       <property name="shortcut">
        <string>Alt+R</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3ButtonGroup</class>
   <extends>Q3GroupBox</extends>
   <header>Qt3Support/Q3ButtonGroup</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local">en61850.h</include>
  <include location="local">widgeom.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>buttonOk</sender>
   <signal>clicked()</signal>
   <receiver>EN61850monbase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonCancel</sender>
   <signal>pressed()</signal>
   <receiver>EN61850monbase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
